cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -65,6 +66,7 @@ func init() {
 
 	flag.Var(utilflag.NewMapStringBool(&featureGates), "feature-gates", "A set of key=value pairs that describe feature gates for alpha/experimental features. "+
 		"Options are:\n"+strings.Join(utilfeature.DefaultFeatureGate.KnownFeatures(), "\n"))
+	flag.Var(utilflag.NewMapStringString(&pvcAnnotationMappings), "pvc-annotation-mappings", "A set of key=value pairs that describe how pvc annotation should be mapped to parameters that are passed to csi drivers.")
```

Comment:
I think this description is a little confusing since its not really `key=value`, it's actually more like `pvcannotationkey=csidriverparameterkey` (<- this isn't actually good I urge you to reword so that this concept is more clear though)